pkgs/applications/video/droidcam/default.nix

```diff
@@ -42,7 +51,10 @@ stdenv.mkDerivation rec {
     runHook preInstall
 
     install -Dt $out/bin droidcam droidcam-cli
-    install -D icon2.png $out/share/icons/hicolor/droidcam.png
```

Comment:
Why not just keep the `install` command? It will create the directory automatically.